### PR TITLE
attempting to add metadata for coanzse.org

### PR DESCRIPTION
**Pull request by stcoats**

### CODE REVIEW

**Comment by andmor-** [comment]
I am a little confused by this. There are [two new files](https://github.com/clarin-eric/SPF-SPs-metadata/pull/275/files), which one is to keep? 

There should only be one file (placed in: `/metadata/coanzse.org.xml`)

**Comment by stcoats** [comment]
I am also confused, and more than a little! 😄 I don't know why there are two files. I think somehow in the process of attempting to follow the bafflingly complex, multi-step, over-many-different pages-distributed instructions for setting up an SPF, I might have mistakenly put a .conf file in the wrong place or used the wrong XPath expression or recreated a cloned directory in the wrong place or attempted to merge the wrong branch with the wrong master or something. I really have no idea. 
Looking in my local CLARIN directory, which I cloned as part of the tutorial instructions, it appears that the file I created in there is 191 lines long. That means it would be the second of the two files. I guess just delete the first one? However, I also vaguely remember part of the tutorial requiring me to make various changes and then to to `rsync` different files between `etc` and the CLARIN cloned directory, or something along those lines? Sorry for not being an expert in this, and thanks for any help you can provide.

**Comment by andmor-** [comment]
The files seem different. I will use the one in `metadata/coanzse.org.xml` for now.
Instead of `rsync` you can just edit the configuration files in place on your server.

File: metadata/coanzse.org.xml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+Uh7phBZwMhOFTTvSK8E6V6NS61IAAAGKGLLGcAAABAMARzBFAiBez/XvF8WNv5gx
+7JZEFkR/EFlJXIKtpvZAv5PnywvF6QIhAO0hclOw7Tkwgugi2G5KpR1kcdeoLPWj
+vsFByac8JttaMA0GCSqGSIb3DQEBCwUAA4IBAQABy8EtdpT1MMbh6j5lLP5cHL+c
+XwgzxZqDXB6tZ8L55bTmQHGwBQRs83mcEp9x0qNq1E/s4fZPpe/IWagO+CNLbbes
+py4I1XtG+wP/HQfzvGO2TAIGT5l+5WqPHqx9328zGmLGcWHOt7rrwNTVduWBuw+H
++xHSjj0kMpDlea6Jc6o7zaQKFEFKXQeUd6vZITax6vAblgiIzJZJmQK2r0Pu/HEI
+SRxyyg7vlfUdhis07KK7snpJeoQzlWxVqx/9RXE+wAezPrkEDWVmD6oOlREcZyRe
+8ferSAK/gNtsv1iZswEOlCmo2Yk3TT4GfxPIW8rz4Qo1Y1zdELpcV6m0eRi8
+</ds:X509Certificate>
+<ds:X509Certificate>
```
Comment (andmor-) [inline metadata/coanzse.org.xml]: There are currently 3 certificates in this file and only one is expected.
We advice you use the same certificate for both: signing and encryption. This certificate should be self-signed (currently it is a letsencrypt signed certificate) and have a validity of 4-5 years.

File: metadata/coanzse.org.xml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+   </md:Extensions>
+   <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
+      <md:Extensions>
+         <mdui:UIInfo>
+            <mdui:DisplayName xml:lang="en">CoANZSE Audio</mdui:DisplayName>
+            <mdui:Description xml:lang="en">CoANZSE Audio Description</mdui:Description>
+            <mdui:InformationURL xml:lang="en">https://coanzse.org</mdui:InformationURL>
+            <mdui:Logo height="220" width="195">https://coanzse.org/corpus-frontend-3.1.0/test/static/img/output-onlinepngtools.png</mdui:Logo>
+            <mdui:Keywords xml:lang="en">Corpus of Australian and New Zealand Spoken English+English+Audio+YouTube Transcripts</mdui:Keywords>
+            <mdui:PrivacyStatementURL xml:lang="en">https://catalog.clarin.eu/privacy_statement.xhtml</mdui:PrivacyStatementURL>
```
Comment (andmor-) [inline metadata/coanzse.org.xml:34]: For testing this will do, but you will need your own Privacy Statement running on you own URLs

**Review by andmor- — CHANGES_REQUESTED** [review]
Please choose(generate) one X509 certificate and replace the 3 ones in your fork by it.

**Comment by stcoats** [comment]
Hi again, I updated the coanzse.org.xml file with the new certificate and attempted a push!

**Comment by andmor-** [comment]
Closing in favor of #277

**Pull request closed without merging**